src/test/lambda/commands/deleteLambda.test.ts

```diff
@@ -6,158 +6,160 @@
-            async () => assert.fail()
-        )
+            // expected results
+            expectDeleteIsCalled: true,
+            expectRefreshIsCalled: true
+        })
+    })
 
-        const node = new MockStandaloneFunctionNode(
-            undefined,
-            parent,
-            {
-                FunctionName: 'myFunctionName'
-            },
-            async () => assert.fail(),
-            async configuration => assert.fail()
-        )
+    it('should not delete lambda when cancelled', async () => {
+        return assertLambdaDeleteWorksWhen({
+            // test variables
+            functionName: 'myFunctionName',
+            onConfirm: async () => false,
 
-        await deleteLambda(node, () => {})
+            // expected results
+            expectDeleteIsCalled: false,
+            expectRefreshIsCalled: false
+        })
+    })
 
-        assert.strictEqual(invokeCount, 1)
+    it('should handles errors gracefully', async () => {
+        const errorToThrowDuringDelete = new SyntaxError('Fake error inserted to test error handling')
+
+        return assertLambdaDeleteWorksWhen({
+            // test variables
+            errorToThrowDuringDelete,
+            functionName: 'myFunctionName',
+            onConfirm: async () => true,
+
+            // expected results
+            expectDeleteIsCalled: true,
+            expectRefreshIsCalled: true,
+            onAssertOutputChannel(outputChannel: MockOutputChannel) {
+                const expectedMessagePart = String(errorToThrowDuringDelete)
+                assert(!outputChannel.isHidden, 'output channel should not be hidden after error')
+                assert(
+                  outputChannel.value && outputChannel.value.indexOf(expectedMessagePart) > 0,
+                  `output channel should contain "${expectedMessagePart}"`
+                )
+            }
+        })
     })
 
-    it('handles errors gracefully', async () => {
-        let deleteCount = 0
-        ext.toolkitClientBuilder = new MockToolkitClientBuilder(
-            undefined,
-            new MockLambdaClient(
-                undefined,
-                async name => {
-                    deleteCount++
-                    throw new Error()
-                }
+    const assertLambdaDeleteWorksWhen = async ({
+        onAssertOutputChannel = ((channel: MockOutputChannel) => {
+            // Defaults to expecting no output. Should verify output when expected.
+            assert.strictEqual(
+              channel.value,
+              '',
+              'expect no output since output testing was omitted'
             )
+        }),
+        ...params
+    }: {
+        functionName: string,
+        errorToThrowDuringDelete?: Error,
+        expectDeleteIsCalled: boolean,
+        expectRefreshIsCalled: boolean,
+        onConfirm(): Promise<boolean>,
+        onAssertOutputChannel?(actualOutputChannel: MockOutputChannel): void
+    }) => {
```

Comment:
Throughout this file, indentation should be 4 spaces.

Reply:
Thanks. It's been a constant battle against my IDE. Need to fix my defaults. Will address the code in a follow on CR.

Reply:
Thanks--please address in this PR rather than a follow-up. Better to avoid checking in misformatted code than to go back and fix it.

Reply:
Done.

Reply:
There are many other places where indentation needs to be fixed. I added comments to each that I saw.